### PR TITLE
berokka - fix broken bioperl vs perl

### PR DESCRIPTION
**Pull request by tseemann**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Perl and Bioperl seem to mistmatched:

```
berokka -h
Can't locate Bio/SeqIO.pm in @INC (you may need to install the Bio::SeqIO module) (@INC contains: /scratch/punim0401/miniconda3/envs/berokka_env/lib/site_perl/5.26.2/x86_64-linux-thread-multi /scratch/punim0401/miniconda3/envs/berokka_env/lib/site_perl/5.26.2 /scratch/punim0401/miniconda3/envs/berokka_env/lib/5.26.2/x86_64-linux-thread-multi /scratch/punim0401/miniconda3/envs/berokka_env/lib/5.26.2 .) at /scratch/punim0401/miniconda3/envs/berokka_env/bin/berokka line 4.
```

### CODE REVIEW

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/berokka/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,37 @@
+{% set name = "berokka" %}
+{% set user = "tseemann" %}
```
Comment (dpryan79) [inline recipes/berokka/meta.yaml]: These are rather excessive unless you plan on renaming the package and transferring ownership.

**Comment by dpryan79** [comment]
Approved so you can merge when you're happy with it.

File: recipes/berokka/meta.yaml (hunk 1 above)
Comment (tseemann) [reply recipes/berokka/meta.yaml]: All my existing packages in bioconda use this.  I understand your argument. I would love to transfer the ownership one day.  Is there any bioconda policy against me doing this?

**Comment by tseemann** [comment]
@dpryan79 is still says it's blocked for review

**Comment by dpryan79** [comment]
Guess I forgot to actually hit approve, sorry about that.

File: recipes/berokka/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/berokka/meta.yaml]: They're your projects, so we certainly have nothing against you transferring them if/when you want :)

**Pull request merged**